Upstream: yes
Reason: fixes error leakage with non GNU tar

From e1b1fd29a781d6f760ba2735e183d03a3e0ea130 Mon Sep 17 00:00:00 2001
From: Kylie McClain <somasis@exherbo.org>
Date: Thu, 28 Apr 2016 00:00:43 -0400
Subject: [PATCH] tar: silence --version's stderr output

Causes an unknown argument error to be output.
---
 completions/tar | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/completions/tar b/completions/tar
index 7b3a486..dc63c12 100644
--- a/completions/tar
+++ b/completions/tar
@@ -697,7 +697,7 @@ _posix_tar()
 _tar()
 {
     local cmd=${COMP_WORDS[0]} func line
-    read line <<<"$($cmd --version)"
+    read line <<<"$($cmd --version 2>/dev/null)"
     case "$line" in
     *GNU*)
         func=_gtar
-- 
2.8.1

